backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/BuyDTO.kt

```diff
@@ -1,9 +1,11 @@
 package rocks.didit.sefilm.domain.dto
 
 import rocks.didit.sefilm.database.entities.ParticipantInfo
+import rocks.didit.sefilm.database.entities.Showing
 import rocks.didit.sefilm.domain.Bioklubbnummer
 
 data class BuyDTO(val bioklubbnummer: Collection<Bioklubbnummer>,
                   val sfBuyLink: String?,
-                  val participantInfo: Collection<ParticipantInfo>) {
+                  val participantInfo: Collection<ParticipantInfo>,
+                  val showing: Showing) {
```

Comment:
Is this strictly necessary? Probably makes the whole serialized object unnecessarily "heavy". Consider sending along only the value you actually need instead of sending the whole `Showing`